streams/src/test/java/org/apache/kafka/test/MockKeyValueStore.java

```diff
@@ -70,18 +57,12 @@ public String name() {
     public void init(final ProcessorContext context,
                      final StateStore root) {
         context.register(root, stateRestoreCallback);
-        if (simulateForwardOnFlush) {
```

Comment:
Piggybacked minor cleanup as the `simulateForwardOnFlush` is not called anymore.